agent/agentbootstrap/bootstrap.go

```diff
@@ -107,7 +107,12 @@ func InitializeState(
 	info.Tag = nil
 	info.Password = c.OldPassword()
 
-	if err := database.BootstrapDqlite(stdcontext.TODO(), database.NewNodeManager(c, logger, coredatabase.NoopSlowQueryLogger{}), logger); err != nil {
+	if err := database.BootstrapDqlite(
+		stdcontext.TODO(),
```

Comment:
OT: It would be good to legitimately work out where to get this context from.

Reply:
It should be the one inside `cmd.Context` that is passed to `Run`.
We just have to thread it through.